device-types/Zyxel/USG-FLEX-200.yaml

```diff
@@ -0,0 +1,43 @@
+---
+manufacturer: Zyxel
+model: USG FLEX 200
+slug: zyxel-usg-flex-200
+u_height: 1.0
+is_full_depth: false
+airflow: passive
+comments: '[Zyxel USG FLEX 200 Firewall](https://www.zyxel.com/global/en/products/next-gen-firewall/usg-flex-firewall-usg-flex-200/license-and-spec)'
```

Comment:
Please add the `weight` and associated `weight_unit` data point if you have it or can find it. Often these are listed in the datasheet, but sometimes they are not

Reply:
done